api/app/models/user.rb

```diff
@@ -99,8 +95,27 @@ def send_welcome_email
     AccountMailer.welcome(self, false).deliver
   end
 
+  def role=(new_role)
+    @pending_role = new_role.to_s
+  end
+
+  def role
```

Comment:
This doesn't seem right, but maybe I'm not understanding why it's here. Didn't we decide to treat this global, calculate role as `kind` rather than role? Seems confusing to give this two names.